lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -389,21 +429,33 @@ def _set_sgi_ip_addr(self, if_name: str):
         except subprocess.SubprocessError as e:
             self.logger.warning("Error while setting SGi IP: %s", e)
 
-    def _is_iface_ip_set(self, if_name, ip_addr):
+    def _is_iface_ip_set(self, if_name, ip_addr, af_inet: int):
         ip_addr = netaddr.IPNetwork(ip_addr)
-        if_addrs = netifaces.ifaddresses(if_name).get(netifaces.AF_INET, [])
+
+        if_addrs = netifaces.ifaddresses(if_name).get(af_inet, [])
 
         for addr in if_addrs:
             addr = netaddr.IPNetwork("/".join((addr['addr'], addr['netmask'])))
             if ip_addr == addr:
                 return True
         return False
 
-    def _flush_ip(self, if_name):
-        flush_ip = ["ip", "addr", "flush", "dev", if_name]
+    def _flush_ip(self, if_name, af_inet: int):
+        ver = "-4"
+        if af_inet == netifaces.AF_INET6:
+            ver = "-6"
+        flush_ip = ["ip", ver, "addr", "flush", "dev", if_name]
         subprocess.check_call(flush_ip)
 
-    def _kill_dhclient(self, if_name):
+    def _cleanup_ipv4_and_ipv6(self, if_name: str):
+        self._cleanup_if(if_name, True, netifaces.AF_INET)
```

Comment:
add keyword args for these
```suggestion
        self._cleanup_if(if_name=if_name, flush=True, af_inet=netifaces.AF_INET)
```